libs/gtkmm2ext/po/fr.po

```diff
@@ -31,27 +32,27 @@ msgstr "Désactivé"
 
 #: action_model.cc:83
 msgid "Main_menu"
-msgstr ""
+msgstr "Menu principal"
```

Comment:
Pas certains que ces choses-là soient à traduire. Où sont-elles affichées pour l'utilisateur ?
Not sure if these things need to be translated. Where are they displayed to the user?

Reply:
Je ne sais pas où est ce qu'elles sont affichées. Ni si il faut les traduire.

Reply:
Astuce de traduction : ne pas traduire les trucs dont on ne sait pas s'ils sont affichés. Car ça risque d'introduire des bogues pour rien.